src/pages/SearchPage.tsx

```diff
@@ -0,0 +1,14 @@
+import { useState } from "react";
+import Navbar from "../components/Navbar";
+import SearchResultsList from "../components/SearchResultsList";
+
+function SearchPage() {
+    const [searchResult, setSearchResult] = useState<string | undefined>()
+    return (
+        <>
+            <Navbar search={true} setSearchResult={setSearchResult}/>
```

Comment:
Why have the prop if you hardcode it to true

Reply:
I did this to create a variation to my Navbar. When the optional search prop is true then the Navbar has a form element, otherwise it only has a button element.

Reply:
Is that used anywhere?
Also please rename it because as a person with no knowledge of that, how would I know what setting that boolean does.
Maybe `showSearchOnNav` or something more obvious